postgres/server/init.sls

```diff
@@ -86,23 +86,43 @@ postgresql-config-dir:
     - require:
       - cmd: postgresql-cluster-prepared
 
-{%- if postgres.postgresconf %}
+{%- set db_port = salt['config.option']('postgres.port') %}
+{%- if db_port %}
+
+postgresql-conf-comment-port:
```

Comment:
> Since the `port` setting is commented in the PG distribution, this is rather safe.

Unfortunately, the `port` is set as part of the installation process, at least on Ubuntu.

So commenting out all and then letting the `file.blockreplace` below do its job.  Works without changes for subsequent runs.

Reply:
Yeah, it appears that `file.comment` state is more intelligent than I thought, and it does not comment out other strings matching same pattern when encounter the first one already commented. Interesting, I was not aware of that...